src/scripts/extensions/authenticationHelper.ts

```diff
@@ -71,6 +72,11 @@ export class AuthenticationHelper {
 				if (isValidUser) {
 					const dataBoundaryHelper = new UserDataBoundaryHelper();
 					let userDataBoundary: string = await dataBoundaryHelper.getUserDataBoundary(response.data);
+					// The default logging has been configured to EU Pipeline. Once we find the
+					// userdataboundary and if it is different from EUDB , reinit the logger with WW Pipeline
+					if (userDataBoundary === DataBoundary[DataBoundary.GLOBAL] || userDataBoundary === DataBoundary[DataBoundary.PUBLIC]) {
+						LogManager.reInitLoggerForDataBoundaryChange(userDataBoundary);
```

Comment:
since this is implemented in web clipper internal, how do we ensure the implementation of this function is also available along with this deployment of clipper? Just wanted to know what happens when the implementation is not available, since build will not fail because we have added this in types (.d.ts file), but will it result in run time crash?

Reply:
Added the function definition locally as well which does not do any reinit as that can happen only in respective tenant environments.